model-client/build.gradle.kts

```diff
@@ -126,3 +130,81 @@ spotless {
         )
     }
 }
+
+val productionLibraryByKotlinOutputDirectory = layout.buildDirectory.dir("compileSync/js/main/productionLibrary/kotlin")
+val preparedProductionLibraryOutputDirectory = layout.buildDirectory.dir("npmPublication")
+
+val patchTypesScriptInProductionLibrary = tasks.register("patchTypesScriptInProductionLibrary") {
```

Comment:
As far as I can tell, this custom tasks is not marked to be cacheable. Any chance to enable caching here to avoid further extending the build times?

Reply:
Thanks for the catch.

I already added `inputs` and `outputs`, but did not enable caching.

Reply:
Done.